src/ripple/app/tx/impl/SetSignerList.cpp

```diff
@@ -330,8 +346,12 @@ SetSignerList::writeSignersToSLE (SLE::pointer const& ledgerEntry) const
 
 // The return type is signed so it is compatible with the 3rd argument
 // of adjustOwnerCount() (which must be signed).
+//
+// NOTE: This way of computing the OwnerCount associated with a SignerList
+// is valid until the featureMultiSignReserve amendment passes.  Once it
+// passes then just 1 OwnerCount is associated with a SignerList.
 int
-SetSignerList::ownerCountDelta (std::size_t entryCount)
+SetSignerList::oldOwnerCountDelta (std::size_t entryCount)
```

Comment:
Calling functions `old...` or `new...` get awkward if we modify this code again. Sometimes during a transition we can name things `old...` and eventually remove the function, but this function needs to stay around forever. For a function like this, I'm OK with a long cumbersome name if you can think of one. Maybe `multiOwnerCountDelta`?

Reply:
`preMultiSignReserveAmendmentOwnerCountDelta`? ;-)

Reply:
Great point about the "old" name.  I struggled with this a bit and then punted.  Maybe `signerCountBasedOwnerCountDelta()`?  That reflects what the function does and what we moved away from.  I'll go with that unless I hear objections or a better suggestion.

Reply:
Yeah, that works :+1: